webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/campaigns/CampaignEventRepository.scala

```diff
@@ -135,7 +139,17 @@ class CampaignEventRepositoryImpl(doobie: Doobie, campaignSerializer: CampaignSe
     val offsetQuery = offset.map(i => fr" offset $i").getOrElse(fr"")
 
 
-    val q = sql"select eventId, campaignId, name, state, startDate, endDate, campaignType from  CampaignEvents " ++ where ++ limitQuery ++ offsetQuery
+    val orderBy = (order,asc) match {
+      case (Some("startDate")|Some("start"), None|Some("asc")) => fr" order by startDate asc"
+      case (Some("startDate")|Some("start"), Some("desc")) => fr" order by startDate desc"
+      case (Some("endDate")|Some("end"), None|Some("asc")) => fr" order by endDate asc"
+      case (Some("endDate")|Some("end"), Some("desc")) => fr" order by endDate desc"
+      case _ => fr""
```

Comment:
There should be a default order chosen by us. I think by start date, newer to older

Reply:
I thought about that too, because no ordering gave results the way they were inserted but i think it has little use